openmc/capi/plot.py

```diff
@@ -218,3 +218,27 @@ def id_map(plot):
     _dll.openmc_id_map(POINTER(_PlotBase)(plot),
                        img_data.ctypes.data_as(POINTER(c_int32)))
     return img_data
+
+
+def property_map(plot):
+    """
+    Generate a 2-D map of (cell_temperature, material_density). Used for
+    in-memory image generation.
+
+    Parameters
+    ----------
+    plot : openmc.capi.plot._PlotBase
+        Object describing the slice of the model to be generated
+
+    Returns
+    -------
+    property_map : numpy.ndarray
+        A NumPy array with shape (vertical pixels, horizontal pixels, 2) of
+        OpenMC property ids with dtype float
+
+    """
+    prop_data = np.zeros((plot.v_res, plot.h_res, 2),
+                         dtype=np.dtype('float'))
+    _dll.openmc_property_map(POINTER(_PlotBase)(plot),
```

Comment:
Declare this function's `argtypes`, `restype`, and `errcheck` up top. After that, you can pass `plot` directly without explicitly converting to a pointer (ctypes knows to do that automatically).

Reply:
Very cool. I'll make that change in `id_map` too